schemas.md

```diff
@@ -91,87 +91,54 @@ The FOIA metadata file should instruct the Platform:
 - what additional fields are required for a perfected request
 - what additional fields the agency would like to collect
 - explanation of additional requirements†
-- FOIA contact information
 
 _† Our research has shown that users often find additional requirements for FOIA
 submission confusing and burdensome. We hope that by having agencies include
 supplemental information about the requirements, it will help inform requesters
 and allay any concerns._
 
-* submission format
-* required form fields
+* agency abbreviation
+* list of departments
```

Comment:
2015 contact data uses the term "departments" but "components" seems more appropriate.